specification/communication/data-plane/Microsoft.CommunicationServicesSms/stable/2021-03-07/communicationservicessms.json

```diff
@@ -0,0 +1,217 @@
+        "enableDeliveryReport": {
+          "description": "Enable this flag to receive a delivery report for this message on the Azure Resource EventGrid.",
+          "type": "boolean",
+          "default": false
+        },
+        "tag": {
+          "description": "Use this field to provide metadata that will then be sent back in the corresponding Delivery Report.",
+          "type": "string"
+        }
+      }
+    },
+    "SendMessageRequest": {
+      "description": "Represents the properties of a send message request.",
+      "required": [
+        "message",
+        "smsRecipient",
+        "from"
+      ],
+      "type": "object",
+      "properties": {
+        "from": {
+          "description": "The sender's phone number in E.164 format that is owned by the authenticated account.",
+          "type": "string"
+        },
+        "smsRecipient": {
+          "description": "The recipient's phone number in E.164 format. In this version, a minimum of 1 and upto 100 recipients in the list are supported.",
+          "maxLength": 100,
+          "minLength": 1,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SmsRecipient"
+          }
+        },
+        "message": {
+          "description": "The contents of the message that will be sent to the recipient. The allowable content is defined by RFC 5724.",
+          "maxLength": 2048,
+          "minLength": 0,
+          "type": "string"
+        },
+        "smsSendOptions": {
+          "$ref": "#/definitions/SmsSendOptions"
+        }
+      }
+    },
+    "SmsSendResponse": {
+      "description": "Response for a successful or multi status send Sms request.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SmsSendResponseItem"
+          }
+        },
+        "nextLink": {
+          "type": "string"
+        }
+      }
+    },
+    "SmsSendResponseItem": {
+      "description": "Response for a single recipient.",
+      "required": [
+        "httpStatusCode",
+        "to",
+        "succeeded"
+      ],
+      "type": "object",
+      "properties": {
+        "to": {
+          "description": "The recipients's phone number in E.164 format.",
+          "type": "string"
+        },
+        "messageId": {
+          "description": "The identifier of the outgoing SMS message. Only present if message processed.",
+          "type": "string"
+        },
+        "httpStatusCode": {
+          "description": "HTTP Status code.",
+          "type": "integer"
+        },
+        "repeatability-result": {
+          "description": "The result of a repeatable request with one of the case-insensitive values accepted or rejected.",
+          "type": "string"
+        },
+        "succeeded": {
+          "description": "Flag to check if message processing succeeded or not.",
```

Comment:
"Indicates if the message is processed successfully or not."

Reply:
Sure